queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
\ No newline at end of file
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+/* List the name of all animals born between 2016 and 2019. */
+SELECT name FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
+
+/* List the name of all animals that are neutered and have less than 3 escape attempts. */
+SELECT name FROM animals WHERE neutered = true AND escape_attempts >= 3;
```

Comment:
👏 Great job on your SQL query! However, there's a minor adjustment needed to meet the project requirement:

- [ ] As per the project requirement, you should answer the query "List the name of all animals that are neutered and have less than 3 escape attempts." Your current query is close, but it retrieves animals with "neutered" set to true and "escape_attempts" of three or more. To align with the project requirement, kindly consider updating your query to include animals with "neutered" status and less than three escape attempts.

By making this adjustment, your query will accurately address the project requirement. Keep up the great work! 👍🐾📋

Reply:
SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;

fixed